v0/destinations/mautic/utils.js

```diff
@@ -0,0 +1,311 @@
+const { CustomError } = require("../../util");
+const { BASE_URL } = require("./config");
+const { httpGET } = require("../../../adapters/network");
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const _ = require("lodash");
+const { getFieldValueFromMessage } = require("../../util");
+const { set } = require("lodash");
+
+// Includes the fields that can be lookUpfields
+const Fields = [
+  "",
+  "id",
+  "title",
+  "role",
+  "firstname",
+  "nps__recommend",
+  "lastname",
+  "cart_status",
+  "company",
+  "sandbox",
+  "car_or_truck",
+  "position",
+  "email",
+  "company_size",
+  "mobile",
+  "prospect_or_customer",
+  "datetime",
+  "phone",
+  "points",
+  "subscription_status",
+  "fax",
+  "b2b_or_b2c",
+  "products",
+  "address1",
+  "address2",
+  "haspurchased",
+  "city",
+  "crm_id",
+  "state",
+  "zipcode",
+  "country",
+  "preferred_locale",
+  "timezone",
+  "last_active",
+  "attribution_date",
+  "attribution",
+  "website",
+  "facebook",
+  "foursquare",
+  "instagram",
+  "linkedin",
+  "skype",
+  "twitter"
+];
+
+const titles = ["Mr", "Mrs", "Miss", "Mr.", "Mrs.", "Miss."];
+
+const isDate = date => {
+  return new Date(date) !== "Invalid Date" && !isNaN(new Date(date));
+};
+const poc = ["Prospect", "Customer"];
+
+const subscription_statuses = ["New", "Existing"];
+
+const roles = [
+  "Individual Contributor",
+  "Manager",
+  "Director",
+  "Executive",
+  "Consultant"
+];
```

Comment:
update variables names in conventional capital letters. i.e, ALLOWED_SUBSCRIPTION_STATUS, ALLOWED_ROLE_VALUES, ALLOWED_TITLES